glide.yaml

```diff
@@ -33,6 +33,8 @@ import:
   version: v1.0.0
 - package: github.com/stretchr/testify
   version: c679ae2cc0cb27ec3293fea7e254e47386f05d69
+- package: github.com/onsi/gomega
```

Comment:
this is pulling in a lot of stuff just to make the test case a little shorter...

Reply:
more like, glide is annoying and can't compute minimal updates :-/

Reply:
Even ignoring stuff other than gomega being pulled in, I meant that pulling in gomega just to use in a few test cases seems excessive. You could just write the test cases not using gomega, like all the existing test cases.

Reply:
Please put glide.yaml changes in a separate commit, then bump in its own commit. Make it easier to review.

Reply:
How about this assert library we use here and there?

Reply:
Do you mean to do `glide update` in commit 1, then edit glide.yaml and `glide update` in commit 2, or edit glide.yaml in commit 1, and `glide update` in commit 2?  Either way, sure I'll use testify/assert.